### PR TITLE
[INFRA-251] unify Dockerfiles across codeboxes

### PR DESCRIPTION
**Pull request by kotchaosu**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,17 @@ FROM ubuntu:trusty
...
+ENV SYNCANO_APIROOT https://api.syncano.io/
+
+RUN groupadd -r syncano && \
+    useradd -u 1000 -r -g syncano syncano -d /tmp -s /bin/bash && \
+    mkdir /home/syncano && \
+    chown -R syncano /home/syncano
+
+# enable everyone to use /tmp
+RUN chmod 1777 /tmp
+# -- CUT --
```
Comment (23doors) [inline Dockerfile]: maybe CUT begin and CUT end would be useful? as there is also a common part at the end

Comment (kotchaosu) [inline Dockerfile]: end is quite tricky because of ENV var after USER only for node, so actually USER is some kind of "CUT END"

File: Dockerfile [hunk 2]
```diff
@@ -2,6 +2,17 @@ FROM ubuntu:trusty
 MAINTAINER "Syncano DevOps Team" <devops@syncano.com>
 
 ENV LAST_REFRESHED 2016-03-25
+ENV SYNCANO_APIROOT https://api.syncano.io/
```
Comment (23doors) [inline Dockerfile:5]: Shouldn't this be after CUT?

Comment (kotchaosu) [inline Dockerfile:5]: this is common part except for Ruby, but I'm trying to contact lib current dev to make it common

File: Dockerfile (hunk 1 above)
Comment (23doors) [inline Dockerfile]: Yeah, but I just thought that we could make it explicit with CUT BEGIN here and CUT END at USER.
About: ENV NODE_PATH /home/syncano/v0.4/node_modules -> we can move this higher above.

Comment (kotchaosu) [inline Dockerfile]: moved up; cut being, cut end... well, possible, but it's six-repos update

**Comment by 23doors** [comment]
looks good!

**Pull request merged**